addons/point_of_sale/models/pos_session.py

```diff
@@ -359,7 +359,17 @@ def _create_account_move(self):
         })
         self.write({'move_id': account_move.id})
 
-        ## SECTION: Accumulate the amounts for each accounting lines group
+        data = {}
+        data = self._accumulate_amounts(data)
+        data = self._create_non_reconciliable_move_lines(data)
+        data = self._create_cash_statement_lines_and_cash_move_lines(data)
+        data = self._create_invoice_receivable_lines(data)
+        data = self._create_stock_output_lines(data)
+        data = self._create_extra_move_lines(data)
```

Comment:
this one doesn't exist in 13.1

Reply:
@nim-odoo Sorry for that!
I had issues with Git and I had to do it twice. The first time I removed the line, the second I forgot... Fixed now.